File: nagasena/test/org/openexi/proc/io/ScriberRuntimeExceptionTest.java
package org.openexi.proc.io;

import java.io.OutputStream;

import junit.framework.Assert;
import junit.framework.TestCase;

public class ScriberRuntimeExceptionTest extends TestCase {

  public ScriberRuntimeExceptionTest(String name) {
    super(name);
  }

  ///////////////////////////////////////////////////////////////////////////
  // Test cases
  ///////////////////////////////////////////////////////////////////////////

  /**
   * Test PREFIX_IS_NULL.
   */
  public void testPrefixIsNull() throws Exception {
    Scriber scriber = new BitPackedScriber(false);
    scriber.setPreserveNS(true);
    try {
      scriber.writePrefixOfQName((String)null, -1, (OutputStream)null);
    }
    catch (ScriberRuntimeException sre) {
      Assert.assertEquals(ScriberRuntimeException.PREFIX_IS_NULL, sre.getCode());
      return;
    }
    Assert.fail();
  }

}

File: nagasenacs/test/Nagasena/Proc/IO/ScriberRuntimeExceptionTest.cs
using System;
using System.IO;
using NUnit.Framework;

namespace Nagasena.Proc.IO {

  [TestFixture]
  public class ScriberRuntimeExceptionTest : Nagasena.LocaleLauncher {

    ///////////////////////////////////////////////////////////////////////////
    // Test cases
    ///////////////////////////////////////////////////////////////////////////

    /// <summary>
    /// Test PREFIX_IS_NULL.
    /// </summary>
    [Test]
    public virtual void testPrefixIsNull() {
      Scriber scriber = new BitPackedScriber(false);
      scriber.PreserveNS = true;
      try {
        scriber.writePrefixOfQName((String)null, -1, (Stream)null);
      }
      catch (ScriberRuntimeException sre) {
        Assert.AreEqual(ScriberRuntimeException.PREFIX_IS_NULL, sre.Code);
        return;
      }
      Assert.Fail();
    }

  }

}